benchmark/resources/utils_ic.js

```diff
@@ -65,7 +66,13 @@ class ICBenchmark extends Benchmark {
         throw new Error(`Invalid model ${err}`);
       }
       rawModel = onnx.ModelProto.decode(loadResult.bytes);
-      importerClass =  OnnxModelImporter;
+      importerClass = OnnxModelImporter;
+    } else if (modelName.indexOf('OpenVino')!== -1) {
+      const networkFile = this.modelInfoDict.modelFile.replace(/bin$/, 'xml');
```

Comment:
`modelInfoDict` now has a new field called `format`, so we may use that to determine format.

Now examples code is no longer depends on the format in `modelName`, like ` (ONNX)`, ` (TFLite)`, ` (OpenVino)`. So when all tests remove the dependencies of format in `modelName`, we could further delete the stuff in the parentheses

Reply:
Thanks @pinzhenx 
I have submitted another patch using 'switch (format)' block.

Reply:
Thanks much. The enhancements you mentioned are helpful.